src/components/App/App.js

```diff
@@ -75,7 +68,7 @@ class App extends Component {
               // logged in shows UserPage else shows LoginPage
               exact
               path="/user"
-              component={UserPage}
+              component={UserPageFuncExport}
```

Comment:
I only added the use of my new component in the direct navigation. All redirects will go to the class component you wrote.